runtime/lua/vim/lsp/util.lua

```diff
@@ -939,33 +944,22 @@ function M.jump_to_location(location)
   -- location may be Location or LocationLink
   local uri = location.uri or location.targetUri
   if uri == nil then return end
+  local bufnr = vim.uri_to_bufnr(uri)
```

Comment:
Again, this seems out of scope for this PR

Reply:
ditto. fixed